kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -96,6 +96,35 @@
   const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
+    $trNameSpace: 'user-edit-modal',
+    $trs: {
+      modalTitle: 'Edit Account Info',
+      // input labels
+      fullName: 'Full Name',
+      username: 'Username',
+      userKind: 'User Kind',
+      enterNewPw: 'Enter new password',
+      confirmNewPw: 'Confirm new password',
```

Comment:
Good old `Title Case` vs. `Sentence case` conundrum... 😒  

We didn't establish any internal conventions, nor I'm aware that we cover this in our Style Guide, yet. 
Both [GNOME](https://developer.gnome.org/hig/stable/writing-style.html.en) and [Apple](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/TerminologyWording.html#//apple_ref/doc/uid/20000957-CH15-SW4) HIG favor the Title Case for action elements, so these should be:

`Enter New Password`
`Confirm New Password`